src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/TagHelperCompletionProvider.cs

```diff
@@ -204,6 +204,7 @@ public override IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorCompl
             var ancestors = containingElement.Ancestors();
             var (ancestorTagName, ancestorIsTagHelper) = _tagHelperFactsService.GetNearestAncestorTagInfo(ancestors);
             var elementCompletionContext = new ElementCompletionContext(
+                containingElement,
```

Comment:
I'm not a big fan of this name as I feel it's ambiguous, but it seems to be the accepted name so I'm going with it.